pkg/neotest/signer.go

```diff
@@ -44,6 +44,14 @@ type MultiSigner interface {
 	Single(n int) SingleSigner
 }
 
+// ContractSigner is an interface for contract signer.
+// `InvocationScript()` and `ScriptHash()` must be defined.
+type ContractSigner interface {
+	Signer
+	// InvocationScript returns an invocation script to sign contract.
```

Comment:
s/`to sign contract`/`to be used as invocation script for contract-based witness`?

Reply:
Done.